### PR TITLE
Use current fiscal period and year when creating reversals

### PR DESCRIPTION
**Pull request by sprucely**
The uglier FiscalYear and FiscalPeriod extension methods weren't actually being used. Moved the cleaner implementation to Sloth.Core.

### CODE REVIEW

File: Sloth.Core/Extensions/DateTimeExtensions.cs [hunk 1]
```diff
@@ -22,54 +22,24 @@ public static DateTime FromPacificTime(this DateTime dateTime)
...
+        /// <returns></returns>
+        public static int GetFiscalPeriod(this DateTime date)
         {
-            if (datetime == null)
-            {
-                throw new ArgumentNullException(nameof(datetime));
-            }
-
-            // Fiscal dates for KFS are PST
-            datetime = datetime.ToPacificTime();
```
Comment (jSylvestre) [inline Sloth.Core/Extensions/DateTimeExtensions.cs:61]: @sprucely Should we keep the PST?

File: Sloth.Core/Extensions/DateTimeExtensions.cs [hunk 2]
```diff
@@ -22,54 +22,24 @@ public static DateTime FromPacificTime(this DateTime dateTime)
...
-
-            // return the calculated period:
-            // Subtract 6 from the current month if 
-            // after June 30th and prior to January 1st;
-            // Otherwise add 6 to the current month.
-            if (month > 6 && month <= 12)
-                return (month - 6);
-
-            return (month + 6);
+            return (date.Month + 5) % 12 + 1;
```
Comment (jSylvestre) [inline Sloth.Core/Extensions/DateTimeExtensions.cs:42]: I think it would be useful to keep the comments.

**Review by srkirkland — APPROVED** [review]
some comments are probably useful but in general this looks like it should fix things so 👍

File: Sloth.Core/Extensions/DateTimeExtensions.cs (hunk 1 above)
Comment (sprucely) [reply Sloth.Core/Extensions/DateTimeExtensions.cs:61]: This extension method wasn't even being used. So I think not.

**Pull request merged**